docs/RELEASE_NOTES.md

```diff
@@ -34,7 +34,7 @@
 [Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/1.5.0)
 
 - 🐞 **Bug Fixes** (1)
-    - **Commons** -  Fixed a bug in the `CustomUpdateActionUtils` which caused to generate unnecessary update actions in `buildNewOrModifiedCustomFieldsUpdateActions`. [#428](https://github.com/commercetools/commercetools-sync-java/issues/428)
+    - **Commons** -  Fixed a bug in the `CustomUpdateActionUtils#buildNewOrModifiedCustomFieldsUpdateActions` which caused to generate unnecessary update actions for null custom field values. #428
```

Comment:
it seems you forgot the add link on markdown for `#428`

Reply:
4c956b8c1c00bcea4a7136330f6d2fe2fdb7e5ef